src/app/counselor/components/utils.js

```diff
@@ -0,0 +1,31 @@
+import * as Yup from 'yup'
+
+export const getCounselorCreationInitialValues = () => ({
+  names: '',
+  paternalSurname: '',
+  maternalSurname: '',
+  email: '',
+  shift: '',
+  gender: '',
+  groups: '',
+})
+
+export const getCounselorCreationValidationSchema = () => Yup.object({
```

Comment:
los nombres no adecuados
```suggestion
export const getCounselorValidationSchema = () => Yup.object({
```